dd4t-api/src/main/java/org/dd4t/core/util/TCMURI.java

```diff
@@ -20,16 +20,22 @@
 import java.text.ParseException;
 import java.util.StringTokenizer;
 
+import static org.dd4t.core.util.TCMURI.Namespace.ISH;
+import static org.dd4t.core.util.TCMURI.Namespace.TCM;
+
 public class TCMURI implements Serializable {
 
+    // Note: this variable left for possible backward compatibility
     public static final String URI_NAMESPACE = "tcm:";
+    protected static final String COLON = ":";
```

Comment:
Remove this useless field. It just doesn't make any sense to name the field after the exact character it is representing. Imagine what happens when you decide to change the character for some reason, and how much value will that following commit bring. For better readability it is not bad to have constants inline in code if they are known and will never changes (which is now the case)